render.go

```diff
@@ -152,7 +151,7 @@ func (ren *RenderComponent) generateBufferContent() []float32 {
 	red := colorR
 	green := colorG << 8
 	blue := colorB << 16
-	alpha := uint32(transparency*255.0) << 24
+	alpha := uint32(ren.Transparency*255.0) << 24
```

Comment:
This reminds me of an issue ... `ren.Transparency` should be private, and using a `set`-function. Not because we like it, but because the `generateBufferContent` has to be called again after changing it.

We could also (instead) rename `generateBufferContent` to `FlushCache`, and require the user to call it whenever he changed things like color, transparency, texture, scale. 

Thoughts @paked @faide?